candy.html

```diff
@@ -44,20 +44,33 @@
       <div class="row align-items-center">
         <div class="col-lg-6">
           <div id="headingGroup" class="text-center text-white d-none d-lg-block mt-5">
-            <h1 id="candySpan">MUSEUM
-              <span>/
-              </span>OF
-              <span>/
-              </span>CANDY
-            </h1>
-
+            <h1 id="candySpan">MUSEUM<span>/</span>OF<span>/</span>CANDY</h1>
```

Comment:
this might be nice because its in one line, but lets keep it in a tree like structure, easier to see parents and children relationships between elements